simulator/tests/toEqualInstruction.d.ts

```diff
@@ -1,5 +0,0 @@
-declare namespace jasmine {
```

Comment:
Fuck this file basically!

Reply:
All that effort and we just kill it...I'm almost sad...

Reply:
Haha, yeah :/